jvm/ml4ir-inference/scripts/generate_model.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Script to train a model on a classification dataset in the python project
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+cd ${DIR}/../../../python
+
+CLASSIFICATION_MODEL=ml4ir/applications/classification
+
+PYTHONPATH=. python ${CLASSIFICATION_MODEL}/pipeline.py \
```

Comment:
PYTHONPATH=. ?

Will this run with a set-up virtualenv?  Or what is it using as its python environment?

Reply:
Yes, this is tailored for virtualenv. 
This PYTHONPATH is supposed to be set in the user environment, but we're not sure it is.

Reply:
How does it know to use the virtualenv?

Reply:
It this current form, the script is "just" calling `python` as you can see.
If you env is correctly setup (bin/activate stuff) this would work.

I expect that this will be the case in the circleCI docker image that would run the test.

Reply:
Not sure, we'll have to check.  We may want to make sure it's been called.